src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,161 @@
+@F-001
+Feature: F-001: Retrieval of metadata from doc-store for given document id
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-001
+  Scenario: Successful retrieval of metadata from doc-store for given document id for caseworker
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [S-001-Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   it is submitted to call the [retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a positive response is received
+    And   the response [contains the metadata for the document uploaded above, along with an HTTP 200 OK]
+    And   the response has all other details as expected
+
+  @S-002
+  Scenario: Successful retrieval of metadata from doc-store for given document id with optional fields linked with a serviceAuthorization token
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request[contains valid data for optional fields i.e. serviceAuthorization token]
+    And   it is submitted to call the [retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a positive response is received
+    And   the response [contains the case detail for the updated case, along with an HTTP 200 OK]
+    And   the response has all other details as expected
+
+  @S-003
+  Scenario: Successful retrieval of metadata from doc-store for given document id with optional fields not linked with a serviceAuthorization token
```

Comment:
service auth token has no affect on the user's ability to retrieve document metadata, so this scenario is not required

Reply:
had removed